src/store/actions/index.js

```diff
@@ -8,26 +8,6 @@ axios.defaults.withCredentials = true;
 const BACKEND_URL =
   process.env.REACT_APP_DEPLOYED_URL || 'http://localhost:5000';
 
-// Authentication
-export const success = (history, jwt) => (dispatch) => {
-  localStorage.setItem('token', jwt);
-  axios
-    .get(`${BACKEND_URL}/api/user`, {
-      headers: {
-        authorization: jwt,
-      },
-    })
-    .then((response) => {
-      if (response.data.user.track === null) {
-        history.push('/onboarding');
-      } else {
-        history.push('/');
-      }
-      toast.success('Welcome to the Lambda Alumni Network!');
-    })
```

Comment:
The welcome message I thought was kind of nice - didn't want to keep it?

Reply:
I didn't think twice about removing this. I'll add it back in!